cpu/cc2538/dev/cc2538-ccm-star.c

```diff
@@ -75,49 +75,18 @@ restore_crypto(uint8_t enabled)
 static void
 set_key(const uint8_t *key)
 {
-  cc2538_aes_128_driver.set_key(key);
+  ccm_star_driver.set_key(key);
```

Comment:
- Thanks: it indeed now builds correctly. I'll look into it.
 - With your patch, it's `cc2538_ccm_star_driver` that enables and disables the crypto engine instead of `cc2538_aes_128_driver`. If you want to keep the crypto engine permanently enabled, you have to call either `crypto_init()` or `crypto_enable()` in some initialization code without calling `crypto_disable()` afterwards as currently done in `contiki-main.c`. But disabling the crypto engine when unused is not an issue (the key store seems to be retained). This optimizes the power consumption, and enabling or disabling it is extremely quick as this is only a matter of 3 modifications of registers changing the active clock domains. Or am I missing something?